runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -713,6 +713,26 @@ public void scrollBy(int x, int y) {
         mContentView.scrollBy(x, y);
     }
 
+    public int computeHorizontalScrollRange() {
```

Comment:
I refered to test case here: xwalk/test/android/core/javatests/src/org/xwalk/core/xwview/test/.

This is different from QA's.  But this raise a good question,  is there any possiblity to align test cases under xwalk/test/android/core/javatests/src/org/xwalk/core/xwview/test/ and QA's test?  @lincsoon  @fujunwei @baleboy